src/components/shared/ManagedTree.js

```diff
@@ -44,10 +44,6 @@ class ManagedTree extends Component {
       expanded: new Set(),
       focusedItem: null
     };
-
-    const self: any = this;
```

Comment:
Self is not used anywhere in this component except for the 2 lines below.
I think this should be `this.setExpanded= this.setExpanded.bind(this);` bc on Line 122 it calls it with `this`
`onExpand: item => this.setExpanded(item, true)`